cli/parse-options.js

```diff
@@ -18,6 +19,9 @@ var log = require('npmlog')
  * passe set the `inMemory: true` option. If it’s not set, leveldown is used
  * with the prefix set to `options.data` + 'data' (`.hoodie/data` by default).
  */
+
+PouchDB.plugin(require('pouchdb-mapreduce'))
+
```

Comment:
remove the `  db: {}` default value defined in line 32. Instead replace it with `  PouchDB: PouchDB`. 

Then above the `if (options.dbUrl) {` block, put in the `PouchDB.plugin(require('pouchdb-mapreduce'))` line, so that the plugin only gets loaded when the server gets started. And then instead of setting `config.db.url = options.dbUrl` we want to set the prefix option on the PouchDB constructor, like so:

```js
config.PouchDB = config.PouchDB.defaults({prefix: options.dbUrl})
```

And then you have to move the logic from the lines 31-58 from `server/index.js` below this block, so that `parseOptions` returns the `config.PouchDB` property will all the right plugins loaded and the right defaults set. 

Makes sense?